echopype/tests/echodata/test_echodata_combine.py

```diff
@@ -185,107 +165,50 @@ def test_combine_echodata(raw_datasets):
-
     for group_name in combined.group_paths:
 
-        # ignore the Platform group for ek60_reversed_time parameters (these are checked elsewhere)
-        if (param_id != "ek60_reversed_time") or (group_name != "Platform"):
-
-            # get all Datasets to be combined
-            combined_group: xr.Dataset = combined[group_name]
-            eds_groups = [
-                ed[group_name]
-                for ed in eds
-                if ed[group_name] is not None
-            ]
-
-            # all grp dimensions that are in all_drop_dims
-            if combined_group is None:
-                grp_drop_dims = []
-                concat_dims = []
-            else:
-                grp_drop_dims = list(set(combined_group.dims).intersection(set(all_drop_dims)))
-                concat_dims = list(set(combined_group.dims).intersection(append_dims))
-
-            # concat all Datasets along each concat dimension
-            diff_concats = []
-            for dim in concat_dims:
-
-                drop_dims = [c_dim for c_dim in concat_dims if c_dim != dim]
-
-                diff_concats.append(xr.concat([ed_subset.drop_dims(drop_dims) for ed_subset in eds_groups], dim=dim,
-                                    coords="minimal", data_vars="minimal"))
-
-            if len(diff_concats) < 1:
-                test_ds = eds_groups[0]  # needed for groups that do not have append dims
-            else:
-                # create the full combined Dataset
-                test_ds = xr.merge(diff_concats, compat="override")
-
-                # correctly set filenames values for constructed combined Dataset
-                if "filenames" in test_ds:
-                    test_ds.filenames.values[:] = np.arange(len(test_ds.filenames), dtype=int)
-
-                # correctly modify Provenance attributes so we can do a direct compare
-                if group_name == "Provenance":
-
-                    if param_id == "ek60_reversed_time":
-                        test_ds.attrs["reversed_ping_times"] = 1
-                    else:
-                        test_ds.attrs["reversed_ping_times"] = 0
-
-                    del test_ds.attrs["conversion_time"]
-                    del combined_group.attrs["conversion_time"]
-
-            if (combined_group is not None) and (test_ds is not None):
-                assert test_ds.identical(combined_group.drop_dims(grp_drop_dims))
-
-    temp_zarr_dir.cleanup()
-
-    # close client
-    client.close()
+        # get all Datasets to be combined
+        combined_group: xr.Dataset = combined[group_name]
+        eds_groups = [
+            ed[group_name]
+            for ed in eds
+            if ed[group_name] is not None
+        ]
+
+        # all grp dimensions that are in all_drop_dims
+        if combined_group is None:
+            grp_drop_dims = []
+            concat_dims = []
+        else:
+            grp_drop_dims = list(set(combined_group.dims).intersection(set(all_drop_dims)))
```

Comment:
This is just a question so that I understand the intention:
`all_drop_dims` basically contains the dimensions that should not be compared at the end between the directly constructed combined dataset (within this test function) and the one created by running `combine_echodata`. Is this correct?

Reply:
Yes, `all_drop_dims` contains all dimensions that should not be compared. Additionally, all variables that contain a dimension in `all_drop_dims` will also be dropped (since I use `ed_subset.drop_dims`). This is necessary because the Dataset created by `combine_echodata` contains additional variables in the `Provenance` group, such as `echodata_filename, platform_attr_key, etc.`.